packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/Modal/ActionBody.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { Loader } from '@strapi/design-system/Loader';
+import { Grid } from '@strapi/design-system/Grid';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+import { pxToRem } from '@strapi/helper-plugin';
+import getDefaultMessage from '../utils/getActionTypesDefaultMessages';
+import ActionItem from './ActionItem';
+
+const ActionBody = ({ status, data, formattedDate }) => {
+  const { formatMessage } = useIntl();
+
+  if (status === 'loading') {
+    return (
+      <Flex padding={7} justifyContent="center" alignItems="center">
+        <Loader />
+      </Flex>
+    );
+  }
```

Comment:
I feel like we are at at stage where we could unify that in the DS. Would you mind creating an issue about the factor we should support a consistent loading state for modals? :)

Reply:
Sure, would you know where I can find more examples of this pattern in the product?

Reply:
- `BulkMoveDialog`
- `EditAssetDialog`

are two that I can think of but there are probably many more :)

Reply:
https://github.com/strapi/design-system/issues/841